src/test/java/seedu/address/testutil/LoanBuilder.java

```diff
@@ -20,19 +21,22 @@
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "alice@gmail.com";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
+    public static final String DEFAULT_LOANSTATUS = "ONGOING";
```

Comment:
Here, you can possibly use the Enum itself instead of a `String`.

```java
public static final LoanStatus DEFAULT_LOANSTATUS = LoanStatus.ONGOING;
```

Won't make a massive difference, but may as well. Makes it easier to catch when renaming `ONGOING` should you need to for some reason.